ps_mbo.php

```diff
@@ -346,7 +346,7 @@ public function installTabs()
      *
      * @return bool
      */
-    public function installTab(array $tabData)
```

Comment:
FYI `array` and class names are possible options on PHP 5. The only ones you have to remove are the rest of primitive types (`string`, `int`)